src/pdn/README.md

```diff
@@ -128,188 +136,205 @@ define_pdn_grid -macro \
 | `-name` | Defines a name to use when referring to this grid definition. |
-| `-voltage_domain` | Defines the name of the voltage domain for this grid. (Default: Last domain created) |
-| `-starts_with` | Specifies whether the first strap placed will be POWER or GROUND (Default: GROUND) |
-| `-grid_over_boundary` | Place the power grid over the entire macro. |
-| `-grid_over_pg_pins` | Place the power grid over the power ground pins of the macro. (Default) |
+| `-grid_over_pg_pins`, `-grid_over_boundary` | Place the power grid over the power ground pins of the macro. (Default True), or Place the power grid over the entire macro.  |
+| `-voltage_domain` | Defines the name of the voltage domain for this grid. (Default: Last domain created). |
+| `-orient` | For a macro, defines a set of valid orientations. LEF orientations (N, FN, S, FS, E, FE, W and FW) can be used as well as standard geometry orientations (R0, R90, R180, R270, MX, MY, MXR90 and MYR90). Macros with one of the valid orientations will use this grid specification. |
 | `-instances` | For a macro, defines a set of valid instances. Macros with a matching instance name will use this grid specification. |
 | `-cells` | For a macro, defines a set of valid cells. Macros which are instances of one of these cells will use this grid specification. |
 | `-default` | For a macro, specifies this is a default grid that can be overwritten. |
-| `-orient` | For a macro, defines a set of valid orientations. LEF orientations (N, FN, S, FS, E, FE, W and FW) can be used as well as standard geometry orientations (R0, R90, R180, R270, MX, MY, MXR90 and MYR90). Macros with one of the valid orientations will use this grid specification. |
-| `-halo` | Specifies the default minimum separation of selected macros from other cells in the design. This is only used if the macro does not define halo values in the LEF description. If 1 value is specified it will be used on all 4 sides, if two values are specified, the first will be applied to left/right sides and the second will be applied to top/bottom sides, if 4 values are specified, then they are applied to left, bottom, right and top sides respectively. (Default: 0) |
-| `-obstructions` | Specify the layers to add routing blockages, in order to avoid DRC violations |
-| `-bump` | Flag to indicate this is a grid for a cover cell |
+| `-halo` | Specifies the default minimum separation of selected macros from other cells in the design. This is only used if the macro does not define halo values in the LEF description. If 1 value is specified it will be used on all 4 sides, if two values are specified, the first will be applied to left/right sides and the second will be applied to top/bottom sides, if 4 values are specified, then they are applied to left, bottom, right and top sides respectively (Default: 0). |
+| `-pins` | Defines a list of layers which where the power straps will be promoted to block pins. |
```

Comment:
```suggestion
| `-halo` | Specifies the design's default minimum separation of selected macros from other cells. This is only used if the macro does not define halo values in the LEF description. If one value is specified, it will be used on all four sides; if two values are specified, the first will be applied to left/right sides, and the second will be applied to top/bottom sides; if four values are specified, then they are applied to left, bottom, right and top sides respectively (Default: 0). |
| `-pins` | Defines a list of layers where the power straps will be promoted to block pins. |
```